src/main/java/pages/Page.java

```diff
@@ -15,4 +29,21 @@ public Page(WebDriver driver) {
     public WebDriver getDriver() {
         return driver;
     }
+
+    public void waitVisibleElement(WebElement elementLocation) {
+        new WebDriverWait(driver, visibilityTimeout).until(ExpectedConditions.visibilityOf(elementLocation));
+    }
+
+    public void waitClickable(final WebElement element) {
+        Wait<WebDriver> wait = new FluentWait<WebDriver>(driver)
+                .withTimeout(timeoutFluent, TimeUnit.SECONDS)
+                .pollingEvery(pollingTime, TimeUnit.SECONDS);
+
+        wait.until(new Function<WebDriver, Boolean>() {
+            public Boolean apply(WebDriver driver) {
+                return element.isDisplayed();
```

Comment:
It's checking for visibility, but not clickability as it is called("waitClickable").
You can rename method or rewrite its logic.